log4j-test-config/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+ambry.log.dir=logs
+log4j.rootLogger=INFO, stdout
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+log4j.additivity.state.change.logger=false
+
+# Define the file appender for Public Access Log
+log4j.appender.PublicAccessLog=org.apache.log4j.FileAppender
+log4j.appender.PublicAccessLog.File=${ambry.log.dir}/publicAccessLog.out
```

Comment:
Is ambry.log.dir always defined?

Reply:
Its defined in this file (first property) - this is just a copy of the log4j props file in config.